src/components/accordion/accordion.njk

```diff
@@ -4,7 +4,7 @@
       <button class="rvt-accordion__toggle" id="{{ id }}-{{ loop.index }}-label" data-rvt-accordion-trigger="{{ id }}-{{ loop.index }}">
         <span class="rvt-accordion__toggle-text">{{ panel.summary }}</span>
         <div class="rvt-accordion__toggle-icon">
-          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+          <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 16 16">
```

Comment:
I think we might need to handle increasing the size of this icon as a separate issue/PR. For whatever reason increasing the size to 20px is causing some "fuzziness" in the rendering. There may just be some jank in the source svg icon. It looks good at 16px which is the size the icons was drawn at, but for some reason at 20px it's not rendering sharply like it should. Screenshots below show the original 16px svg on the top vs. the upsized 20px svg on the bottom. The rendering is pretty fuzzy.

Let's set the `width` and `height` attributes back to `16` for now and I can create a separate issue/task to address this. We may just need to create a custom icon that is dialed in to work at this size, or redraw the plus icon from the rivet icons set.

![image](https://user-images.githubusercontent.com/5590034/155555136-ec50347d-a76d-4d50-8d30-8e3bb869bcdd.png)

![image](https://user-images.githubusercontent.com/5590034/155554978-c37cc1be-3298-4e74-bc3a-c9b2f136dd87.png)